waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -83,9 +84,18 @@ genMigrationsDir spec =
 -- | This function operates on generated code, and thus assumes the file drafts were written to disk
 postWriteDbGeneratorActions :: AppSpec -> Path' Abs (Dir ProjectRootDir) -> IO ([GeneratorWarning], [GeneratorError])
 postWriteDbGeneratorActions spec dstDir = do
+  dbEnvironmentError <- maybeToList <$> checkForDbEnvVars spec 
   dbGeneratorWarnings <- maybeToList <$> warnIfDbNeedsMigration spec dstDir
   dbGeneratorErrors <- maybeToList <$> genPrismaClient spec dstDir
-  return (dbGeneratorWarnings, dbGeneratorErrors)
+  return (dbGeneratorWarnings, dbGeneratorErrors ++ dbEnvironmentError )
+
+checkForDbEnvVars :: AppSpec -> IO (Maybe GeneratorError)
```

Comment:
Hi Martinsos, thank you for the feedback.  I converted the PR to a draft because I can't get my current code to compile and I'm not yet sure how to resolve the problem I'm running into.  In order to check that the `DATABASE_URL` environment variable is set I need to call `lookupEnv` which has the type signature of `lookupEnv:: String -> IO (Maybe String)`.  Am I correct in thinking that I need to add `IO` to the signature of  `genPrismaSchema :: AppSpec -> Generator FileDraft` ?